src/resources/views/list.blade.php

```diff
@@ -10,7 +10,8 @@
         </div>
         <div class="panel-body">
             <form role="form" action="{{ route('teamspeak.add') }}" method="post">
-                {{ csrf_field() }}
+            <input type="hidden" name="_token" value="{{ csrf_token() }}" />
```

Comment:
best to use `csrf_field` helper, so we don't have to post an hotfix if framework change something in the way csrf is handled.